Templates/JenkinsPipeline/Jenkinsfile

```diff
@@ -411,6 +426,16 @@ pipeline {
 						
 							dir ("${BuildOutputDir}") {
 								BuildFile = findFiles(glob: "buildList.txt")
+								TazReports = findFiles(glob: TazLogs)
```

Comment:
1. For better readability I would prefer to see the publishing after the build status is reported to the build.
2. Changing my mind, we should introduce a variable `tazReportsDoPublish` to let the user turn on the publishing via this flag. Please also add the variable to the prints of the build variable summary.